### DISCUSSION TITLE
javascript error: javascript error: Object.hasOwn is not a function

### DISCUSSION THREAD

**Discussion by iniyavans** [webdriverio/webdriverio#11725] [Community]
```
const isGetStartedDisplayed = await this.getingStarted.isDisplayed();
console.log(`The value is ${isGetStartedDisplayed}`);
await this.getingStarted.click();
```

from the above code I got true from isGetStartedDisplayed. But the js throw

javascript error: javascript error: Object.hasOwn is not a function
  (Session info: chrome=119.0.6045.159)
  
above error with a click function. Anyone can please assist me resolve this issue?

**Comment by christian-bromann** [comment]
@iniyavans can you provide the whole stack trace please?

**Reply by iniyavans** [reply]
Kindly checkout the below image @christian-bromann 
![image](https://github.com/webdriverio/webdriverio/assets/88368294/9dc5418c-4cfe-41e0-b3a7-7850d7ea37e1)

**Reply by christian-bromann** [reply]
Can you also provide Chromedriver logs please?

**Reply by iniyavans** [reply]
@christian-bromann I'm New here, its possible to guide me to get the log?

**Reply by iniyavans** [reply]
Do you mean this?
![Screenshot_2023-11-27_14_16_24](https://github.com/webdriverio/webdriverio/assets/88368294/5db17608-2229-4254-8d61-ae63b3ba9b67)

**Reply by christian-bromann** [reply]
No, set the `outputDir` option in your `wdio.conf.js` and you should get a `wdio-0-0-chromedriver.log` file. Please provide the contents of that. Or if possible a minimal reproducible example.

**Reply by iniyavans** [reply]
[wdio-0-0-chromedriver.log](https://github.com/webdriverio/webdriverio/files/13477458/wdio-0-0-chromedriver.log)
I'll attach the file. But I can't get any errors there. @christian-bromann

**Reply by christian-bromann** [reply]
Can you provide a reproducible example then? I checked the code base and there is no usage of `Object.hasOwn` which makes me thing it is not related to WebdriverIO itself.

**Reply by iniyavans** [reply]
@christian-bromann I'll push the code and share with you? I think its better to reproduce the issue from your side.

**Reply by christian-bromann** [reply]
I am not asking for your project code, I am asking for a new minimal project that helps to reproduce this. No maintainer will spend hours trying to reproduce an issue that is potentially only on the reporters side. I don't know what environment and what type of application you are running.

**Reply by iniyavans** [reply]
Ok, @christian-bromann Got it. 

I'll be running this code on the Metamask extension. I'll just launch the browser switch to the metamask tab and then try to click the button. At that time I'm getting an Object.hasOwn is not a function error. But the setValue is working fine.

**Reply by iniyavans** [reply]
`get getingStarted() {
        return $('//button');
    }

async login() {

        await browser.url('https:google.com');

        await browser.maximizeWindow()

        // Wait until there are 2 open windows or a timeout occurs
        await browser.waitUntil(async () => {
            const handles = await browser.getWindowHandles();
            return handles.length === 2;
        }, {
            timeout: 15000,
            timeoutMsg: 'Expected 2 windows to be open within 15 seconds',
            interval: 1000,
        });

        await browser.switchWindow('nkbihfbeogaeaoehlefnkodbefgpgknn');

        // Check if the 'getingStarted' button is displayed
        const isGetStartedDisplayed = await this.getingStarted.isDisplayed();

        console.log(`The value is ${isGetStartedDisplayed}`);

        await this.getingStarted.click();

    }`
    
    This is the code what I used to @christian-bromann

**Reply by christian-bromann** [reply]
This doesn't help me to reproduce the issue, can you reproduce it outside of a Chrome extension? It is likely a Chromedriver bug and I recommend to raise an issue in https://bugs.chromium.org/p/chromedriver/issues/list

**Reply by iniyavans** [reply]
The click function is working fine to the outside of the Chrome extension. But the same function is working on Selenium. I can click the elements inside the Chrome extension with a Selenium. @christian-bromann

**Reply by christian-bromann** [reply]
Can you provide Chromedriver logs when run via WebdriverIO vs Selenium. I would be curious to see how it differs. Again, this bug is  unlikely related to WebdriverIO maintained code so I am expecting Selenium to run into similar issues.
